### DISCUSSION TITLE
ignore this its for video tutorial

### DISCUSSION THREAD

**Discussion by Gosto70** [community/community#136185] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Select Topic Area
Question

Body
Top area
its body discussion
sorry this its not for discussion its video tutorial

**Comment by wechuck** [comment]
answered two

**Comment by thiagosousai** [comment]
Response #1

**Comment by Dhrubojyot** [comment]
Response #1

**Comment by ebndev** [comment, marked as answer]
Hi @Gosto70 & @wechuck, thanks for being a part of the GitHub Community!

While we appreciate you wanting to contribute to Discussions, please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this, and other related posts. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
